IntegrationTests/BraintreePayPal_IntegrationTests.swift

```diff
@@ -14,8 +14,8 @@ class BraintreePayPal_IntegrationTests: XCTestCase {
             return
         }
         
-        let payPalClient = BTPayPalClient(apiClient: apiClient)
-        
+
+        let payPalClient = BTPayPalClient(authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn")
```

Comment:
Can we extract `sandbox_9dbg82cq_dcpspy2brwdjr3qn` into a constant `authorization` to be used in this file?

Reply:
yep, updated in 0ff787945f1b13c6a05416f87ac29cb794dc31a0